docs/guide/index.md

```diff
@@ -95,26 +95,27 @@ vitest related /src/index.ts /src/hello-world.js
 
 ### CLI Options
 
-| Options       |               |
-| ------------- | ------------- |
-| `-v, --version` | Display version number |
-| `-r, --root <path>` | Define the project root |
-| `-c, --config <path>` | Path to config file |
-| `-u, --update` | Update snapshots |
-| `-w, --watch` | Watch mode |
-| `-o, --open` | Open UI (default: false) |
+| Options                           |               |
+|-----------------------------------| ------------- |
+| `-v, --version`                   | Display version number |
+| `-r, --root <path>`               | Define the project root |
+| `-c, --config <path>`             | Path to config file |
+| `-u, --update`                    | Update snapshots |
+| `-w, --watch`                     | Watch mode |
+| `-o, --open`                      | Open UI (default: false) |
 | `-t, --testNamePattern <pattern>` | Run tests with names matching the pattern |
-| `--api` | Listen to port and serve API |
-| `--threads` | Enable Threads (default: true) |
-| `--silent` | Silent console output from tests |
-| `--reporter <name>` | Select reporter: `default`, `verbose`, or `dot` |
-| `--coverage` | Use c8 for coverage |
-| `--run` | Do not watch |
-| `--global` | Inject APIs globally |
-| `--dom` | Mock browser api with happy-dom |
-| `--environment <env>` | Runner environment (default: node) |
-| `--passWithNoTests` | Pass when no tests found |
-| `-h, --help` | Display available CLI options |
+| `--api`                           | Listen to port and serve API |
+| `--host <host>`                   | Specify which IP addresses the serve API should listen on. Set to 0.0.0.0 to listen on all addresses, including LAN and public addresses. |
```

Comment:
shouldn't this just be "Specify which IP address ....." instead of "addresses"?

Reply:
upps, just copy/paste from `vite cli` docs

Reply:
@patak-dev can we specify a list on vite?

Reply:
I think it is "addresses" because "Set to 0.0.0.0 to listen on all addresses, including LAN and public addresses."